src/crimson/os/seastore/seastore.cc

```diff
@@ -1385,6 +1403,68 @@ SeaStore::tm_ret SeaStore::_setattrs(
     std::move(aset));
 }
 
+SeaStore::tm_ret SeaStore::_rmattr(
+  internal_context_t &ctx,
+  OnodeRef &onode,
+  std::string name)
+{
+  LOG_PREFIX(SeaStore::_rmattr);
+  DEBUGT("onode={}", *ctx.transaction, *onode);
+  auto& layout = onode->get_mutable_layout(*ctx.transaction);
+  if (name == OI_ATTR) {
```

Comment:
If layout.oi_size == 0, it's possible that OI_ATTR is set, but too large for this storage.  This condition needs to be

```
if ((name == OI_ATTR) && (layout.oi_size > 0)) {
```

Ditto below for SS_ATTR

Reply:
test_seastore should have test cases for
1) create OI_ATTR with len > onode_layout_t::MAX_OI_LENGTH, rm OI_ATTR
2) create OI_ATTR with len <= onode_layout_t::MAX_OI_LENGTH, rm OI_ATTR
3) create SS_ATTR with len > onode_layout_t::MAX_SS_LENGTH, rm SS_ATTR
4) create SS_ATTR with len <= onode_layout_t::MAX_SS_LENGTH, rm SS_ATTR

since have size sensitive special cases for those two attrs.

Reply:
done.